CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.23)
 project("FLUF" LANGUAGES C CXX)
 
+set(BINARY_COPY_DESTINATION "C:/Users/josem/Desktop/HomeworldRiseofHiigara/Rise of Hiigara/EXE")
```

Comment:
Remove this hardcoded path please, the BINARY_COPY_DESTINATION variable should be in build.conf